spec/src/main/asciidoc/core/implementation.asciidoc

```diff
@@ -20,8 +20,8 @@ A _managed bean_ is a bean that is implemented by a Java class.
 This class is called the _bean class_ of the managed bean.
 The basic lifecycle and semantics of managed beans are defined by the Managed Beans specification.
 
-
-If a managed bean has a non-static public field, it must have scope `@Dependent`. If a managed bean with a non-static public field declares any scope other than `@Dependent`, the container automatically detects the problem and treats it as a definition error.
+If a managed bean has a non-static public field, its scope must be a pseudo-scope.
```

Comment:
We could, but I figured it would be better to specify what implementations actually do.

Listing `@Dependent` and `@Singleton` as examples, as well as linking the part of the spec that explain what a pseudo-scope is, are good ideas, will do.

Reply:
Done.